frontend/Caddyfile

```diff
@@ -36,7 +36,7 @@
 		Cache-Control "no-store, no-cache, must-revalidate, proxy-revalidate"
 		X-Content-Type-Options "nosniff"
 		Strict-Transport-Security "max-age=31536000"
-		Content-Security-Policy "default-src 'self' https://spt.apps.gov.bc.ca data:; script-src 'self' 'unsafe-eval' https://www2.gov.bc.ca ;style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://use.fontawesome.com; font-src 'self' https://fonts.gstatic.com; img-src 'self' data: https://fonts.googleapis.com http://www.w3.org https://*.gov.bc.ca"
+		Content-Security-Policy "default-src 'self' https://spt.apps.gov.bc.ca data:; script-src 'self' 'unsafe-eval' https://www2.gov.bc.ca ;style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://use.fontawesome.com https://stackpath.bootstrapcdn.com; font-src 'self' https://fonts.gstatic.com https://stackpath.bootstrapcdn.com; img-src 'self' data: https://fonts.googleapis.com http://www.w3.org https://*.gov.bc.ca; connect-src 'self' http://localhost:8000 https://*.gov.bc.ca; child-src 'self' *.gov.bc.ca; worker-src 'self' https://*.gov.bc.ca http://localhost:3005 blob:;"
```

Comment:
blob: here has some security implications so we should scale this back after we get things up and running.